### DISCUSSION TITLE
被钓鱼了

### DISCUSSION THREAD

**Discussion by ParanoidHikari** [share121/inter-knot#5189] [灌水]
上一次看绳网还是好几周前，感觉乱乱的就跑了，昨天愤怒对线完之后注册账号登了进来，一眼看到二维码学习资料，火气很大地举报了
然后发现中计了😡
睡不着，想起来就火大😡

**Comment by Yizakl** [comment]
难绷

**Reply by KELERSAMA** [reply]
Da Lao <img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >

**Comment by miheihuyou** [comment]
哈哈哈哈，前来嘲笑

**Reply by ParanoidHikari** [reply]
别笑了，没脸见人了😡

**Comment by fzdqz** [comment]
哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈就哈哈哈我们钓鱼佬永不空军😂😂😂😂

**Comment by SA0330** [comment]
不会是这个吧
![Uploading Screenshot_20240920_211514_com.tencent.mm.jpg…]()

**Comment by miheihuyou** [comment]
哈哈哈哈，狠狠嘲笑

**Reply by ParanoidHikari** [reply]
你已经笑过了啊，可恶

**Comment by bbgjbjvg** [comment]
哈哈哈😁

**Comment by cuijiaqi0406** [comment]
哈哈哈，我被钓了之后也成为了一个钓鱼佬
